docs/book_files/csi_block_storage_kc_whatsnew.md

```diff
@@ -1,8 +1,8 @@
 # What's new
 
-This topic lists the dates and nature of updates to the published information of IBM® block storage CSI driver 1.11.0.
+This topic lists the dates and nature of updates to the published information of IBM® block storage CSI driver 1.11.1.
 
-|Date|Nature of updates to the published information|
-|----|----------------------------------------------|
-|23 January 2023|The version information was added to IBM Documentation.<br>For more information about this version, see the [What's new in 1.11.0](../content/release_notes/whats_new.md) section in the release notes.|
+| Date       | Nature of updates to the published information                                                                                                                                                          |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 8 May 2023 | The version information was added to IBM Documentation.<br>For more information about this version, see the [What's new in 1.11.1](../content/release_notes/whats_new.md) section in the release notes. |
```

Comment:
why the spaces diff? is it better this way? is it gonna effect the GUI look?

Reply:
According to file formatting rules, the last change is as expected